app/models/concerns/expirable.rb

```diff
@@ -4,6 +4,7 @@ module Expirable
   extend ActiveSupport::Concern
 
   def expires_at
+    return nil if is_a?(Teacher)
```

Comment:
Nice that this works, could we then add to the `Teacher` class:

```rb
def expires_at
  nil
end
```

To overwrite this method and then we don't need a special case in the concern.